test/engine/test_execute.py

```diff
@@ -130,6 +130,13 @@ def test_raw_positional_invalid(self, connection):
             [[3, "ed"], [4, "horse"]],
         )
 
+    def test_create_engine_url_invalid(self):
```

Comment:
this probably should be moved to test/engine/test_parseconnect.py

Reply:
Done